integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/RunCucumberTest.java

```diff
@@ -11,5 +11,6 @@
 @IncludeEngines("cucumber")
 @SelectClasspathResource("features")
 @ConfigurationParameter(key = PLUGIN_PROPERTY_NAME, value = "pretty")
+@ConfigurationParameter(key = PLUGIN_PROPERTY_NAME, value = "com.epam.reportportal.cucumber.ScenarioReporter")
```

Comment:
It looks like the previous `pretty` logger is ignored, see the PR check log.
Can you try to use `@ConfigurationParameter(key = PLUGIN_PROPERTY_NAME, value = "pretty,com.epam.reportportal.cucumber.ScenarioReporter")` and check that it works?

Reply:
@sutaakar 
Thank you for your tip. I've updated the PR and both seems working.